frontends/web/src/hooks/keyboard.test.ts

```diff
@@ -14,24 +14,21 @@
  * limitations under the License.
  */
 
-import { useEffect, useState } from 'react';
+import { fireEvent } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+import { useEsc } from './keyboard';
+import { act } from 'react-dom/test-utils';
 
-const getMatches = (query: string): boolean => window.matchMedia(query).matches;
+describe('useEsc', () => {
 
-export const useMatchMedia = (query: string): boolean => {
-  const [matches, setMatches] = useState<boolean>(getMatches(query));
+  it('should fire its callback when escape key gets pressed', () => {
+    const mock = jest.fn();
+    renderHook((() => useEsc(mock)));
+    act(() => {
+      fireEvent.keyDown(document, { key: 'Escape', code: 27 });
+    });
+    expect(mock).toHaveBeenCalled();
+  });
 
-  useEffect(() => {
-    const handleChange = () => {
-      setMatches(getMatches(query));
-    };
-    const matchMedia = window.matchMedia(query);
-    handleChange();
-    matchMedia.addEventListener('change', handleChange);
-    return () => {
-      matchMedia.removeEventListener('change', handleChange);
-    };
```

Comment:
I'm unsure why this file is marked as moved/renamed when it's not related at all. But here, I created a test file for keyboard hook, and removed useMatchMedia as it's unused. :)